train.py

```diff
@@ -98,7 +100,7 @@
     RuntimeError("EnzPredDataModule not implemented yet")
 else:
     config.classify = True
-    config.watch_metric = "val/aupr"
+    config.watch_metric = "val/f1"
```

Comment:
I initially set this to `aupr` because they use the model with the highest `val/aupr`